tests/e2e/mapper/bluetooth/bluetooth_suite_test.go

```diff
@@ -0,0 +1,287 @@
+/*
+Copyright 2019 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package bluetooth
+
+import (
+	"bytes"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"runtime"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	v1 "k8s.io/api/apps/v1"
+	metav1 "k8s.io/api/core/v1"
+
+	"github.com/kubeedge/kubeedge/cloud/pkg/devicecontroller/apis/devices/v1alpha1"
+	"github.com/kubeedge/kubeedge/tests/e2e/utils"
+)
+
+//context to load config and access across the package
+var (
+	ctx          *utils.TestContext
+	cfg          utils.Config
+	nodeSelector string
+	nodeName     string
+)
+
+var (
+	MockHandler         = "/apis/devices.kubeedge.io/v1alpha1/namespaces/default/devicemodels"
```

Comment:
I think these are constants ?

Reply:
please export only those constants which will be accessed outside this package.

Reply:
Updated